components/custom/game-of-life.tsx

```diff
@@ -0,0 +1,194 @@
+"use client";
+import React, { useState, useEffect, useRef, useCallback } from "react";
```

Comment:
Using `* as React` format here ensures consistency across the project.

Reply:
done